samples/go/README.md

```diff
@@ -1,4 +1,37 @@
 # MIRACL Trust Go OIDC Integration Sample
 
-This example integrates the [go-oidc library](https://github.com/coreos/go-oidc)
-with the MIRACL Trust platform.
+![go](https://github.com/miracl/oidc-samples/workflows/go/badge.svg)
+
+This is an example of an [OIDC integration](https://miracl.com/resources/docs/guides/authentication/oidc/)
+with the [MIRACL Trust platform](https://miracl.com) using Golang.
+
+# Dependencies
+
+This sample uses [go-oidc](https://github.com/coreos/go-oidc) library
+to integrate with the MIRACL Trust platform.
```

Comment:
The OIDC client library is the most important information here. I don't think we should hide it under "Dependencies".